query/gizmo/gizmo_test.go

```diff
@@ -77,165 +77,165 @@ var testQueries = []struct {
 	{
 		message: "get a single vertex",
 		query: `
-			g.V("<alice>").All()
+			g.v("<alice>").all()
```

Comment:
I like this change in general, but the problem is, it will break existing clients.

What do you think about allowing both `V()` and `v()` for now? As I can see from the mapper API, it's only possible to map to a single name. We can work around it by duplicating each method with a special prefix, let's say `X_v` will map to `V`.

Reply:
I was thinking about this as well. Is there any way to add metadata to a function? If not I think a prefix will be okay.

Reply:
No way to add metadata, unfortunately.

Reply:
So I think a prefix will do

Reply:
Made a prefix of `New` for function names.